### ISSUE TITLE
Add back full path to file browser

### ISSUE THREAD

**Issue by ktmriki** [sergey-visual-studio/dpack#26]
Hi!
In the original DPack, in the File Browser you could search the full path of files, like folder\folder\file.cs, for example. I really miss this feature because our codebase has a lot of files that have the same name, but are in different directories.

Thanks a lot!

**Comment by sergey-visual-studio** [comment]
Agreed. How do you used search for it? Give me an example please. Thanks.

**Comment by ktmriki** [comment]
Here's an example:
You have 3 folders, with a file with the same filename in each of them, like so:
`
Folder1\File.cs
Folder2\File.cs
Folder3\File.cs
`

In the old DPack you could just start typing "Folder2\", and it would give you all the files contained in Folder2 (including the proper File.cs).

**Comment by jbalinski** [comment]
Would love to see this added back as well for the same reasons listed above.

**Comment by sergey-visual-studio** [comment]
My recollection of why it was removed from the latest version - determining relative project path had issues for some project types, like web ones opened via URL. And that's just 1 type I recall. W/o relative project path, and lets assume that's not there, how do you see that working if you had folder structure with identical folder names but in different locations of the hierarchy? For instance:

<Base Path>\Folder1\Folder2\file.cs
<Base Path>\Folder2\Folder1\file.cs

Searching for Folder1, should it consider just the last folder in the path or anywhere in the path? If base path happened to contain the same searched folder but not in the relative path, should that be included as well? All sounds pretty messy to me.

**Comment by jbalinski** [comment]
Previously, you could type in any part of file path and name (in any order) and filter down the results.  You could add any number of words separated by spaces.  Here are some examples:

- "file 1" -> would match both files
- "Folder1 file" -> would match both files

For our projects, a more realistic scenario would be:
\ModuleA\file.cs
\ModuleB\file.cs

- "file A" -> would return the first file only
- "file B" - > would return the second file only

The second part of making this useful is allowing users to show the relative paths of the files as a column in the results.  Then in cases where multiple files share the same name and are matched, it is easy to differentiate between the different files.
